src/Job/JobId.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace SlmQueueBeanstalkd\Job;
+
+use Pheanstalk\Contract\JobIdInterface;
+
+class JobId implements JobIdInterface  {
```

Comment:
Formatting is of here. You can use https://github.com/JouwWeb/SlmQueue/blob/master/phpcs.xml to auto format your files if you want.